core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/executionrequest/ExecutionRequestController.java

```diff
@@ -71,16 +71,17 @@
                                                   @RequestParam(required = false, name = "version") String version,
                                                   @RequestParam(required = false, name = "environment") String environment,
                                                   @RequestParam(required = false, name = "label") String labelKeyCombination,
-                                                  @RequestParam(required = false, name = "run-id") String runId) {
-        List<ExecutionRequestFilter> executionRequestFilters = extractScriptFilterOptions(script, version, environment, labelKeyCombination, runId);
+                                                  @RequestParam(required = false, name = "run-id") String runId,
+                                                  @RequestParam(required = false, name = "runStatus") String runStatus) {
```

Comment:
for urls we use the convention of: hyphenated separation of the words. So: run-status